docs/contents/user-docs/irsa.md

```diff
@@ -1,58 +1,31 @@
+
+By using the IRSA feature, you no longer need to provide extended permissions to the node IAM role so that pods on that node can call AWS APIs. You can scope IAM permissions to a service account, and only pods that use that service account have access to those permissions. 
 
-    Note that you do *not* need to be using the Amazon EKS service in order to
-    use IRSA. There are [instructions][1] on the
-    amazon-eks-pod-identity-webhook repository for setting up IRSA on your own
-    Kubernetes installation.
+!!! note "EKS is not required to use IRSA"
+    You do not need to use the Amazon EKS service in order to use IRSA. You can [set up IRSA on your own Kubernetes installation][1].
 
-## IRSA setup on EKS cluster and install ACK controller using Helm
-Following steps provide example to setup IRSA on EKS cluster to install ACK ElastiCache controller using Helm charts.
-By modifying the variables values as needed, these steps can be applied for other ACK controllers.
+The following steps demonstrate how to set up IRSA on an EKS cluster while installing the ACK ElastiCache controller using Helm charts. By modifying the variable values as needed, these steps can be applied for the installation of other ACK service controllers.
 
-The steps include:
+## Step 1. Create an OIDC identity provider for your cluster
 
-### 1. Create OIDC identity provider for cluster
-Create OIDC identity provider for cluster using CLI command.
-Example:
+Create an [OpenID Connect (OIDC) identity provider][9] for your EKS cluster using the `eksctl utils` command:
 ```
-EKS_CLUSTER_NAME=<eks cluster name>
+export EKS_CLUSTER_NAME=<eks cluster name>
 eksctl utils associate-iam-oidc-provider --cluster $EKS_CLUSTER_NAME --approve
 ```
-For detailed instructions, follow [Enabling IAM roles for service accounts on your cluster][3].
+For detailed instructions, refer to Amazon EKS documentation on how to [create an IAM OIDC provider for your cluster][3].
 
-### 2. Create an IAM role and policy for service account
-For detailed instructions, follow instructions at [Creating an IAM role and policy for your service account][4].
+## Step 2. Create an IAM role and policy for your service account
 
-#### 2(a) - Create IAM role
+### Create an IAM role for your ACK service controller
 ```
 AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
-OIDC_PROVIDER=$(aws eks describe-cluster --name $EKS_CLUSTER_NAME --query "cluster.identity.oidc.issuer" --output text | sed -e "s/^https:\/\///")
+OIDC_PROVIDER=$(aws eks describe-cluster --name $EKS_CLUSTER_NAME --region <eks cluster region> --query "cluster.identity.oidc.issuer" --output text | sed -e "s/^https:\/\///")
```

Comment:
If we provide region here, we should be providing it in the previous `eksctl` command? Both or neither, I don't mind either way.

Reply:
Got an error that I needed to specify region when I went through this, so I'll add to both.